core/src/services/gcs/core.rs

```diff
@@ -208,7 +208,12 @@ impl GcsCore {
         }
     }
 
-    pub async fn gcs_get_object_metadata(&self, path: &str) -> Result<Response<IncomingAsyncBody>> {
+    pub async fn gcs_get_object_metadata(
+        &self,
+        path: &str,
+        if_match: Option<&str>,
```

Comment:
Maybe we can pass `args: Option` directly for shore code (could be done in seperate PR)

Reply:
We may consider a normal solution for increasing parameters.

Reply:
I think passing `OpStat` is normal enough, no extra params will needed any more(

Reply:
We could have a PR for migrating this situation all at once.